src/mscorlib/src/System/AppDomain.cs

```diff
@@ -330,18 +330,12 @@ public String BaseDirectory
             }
         }
 
-        public override String ToString()
+        public override string ToString()
```

Comment:
This is unused - AppDomain in CoreLib is internal. The actual AppDomain type lives in CoreFX.

You can delete this method.

Reply:
IsFullyTrusted property seems to be unused as well.